iati/core/__init__.py

```diff
@@ -2,4 +2,4 @@
 from .codelists import Code, Codelist
 from .data import Dataset
 from .rulesets import Rule, Ruleset
-from .schemas import Schema
+from .schemas import ActivitySchema, OrganisationSchema, Schema
```

Comment:
This is resolved for `Schema` in 2219c58. I'll leave for you and @allthatilk to take this forward for `Rule` (and one for us to note when we're doing PR reviews for those branches).